src/js/index.js

```diff
@@ -0,0 +1,83 @@
+const numberBtn = document.getElementById("number");
+const totalCount = document.getElementById("total");
+const clearBtn = document.getElementById("clear");
+const operatorBtn = document.getElementById("operator");
+
+const totalCountToArr = (unit) => totalCount.innerHTML.split(unit);
+const isOperator = () => isNaN(Number(value));
+
+const isMaxLength = () => {
+  const curOperator = totalCountToArr("").find((v) => isOperator(v));
+  if (curOperator) {
+    return totalCountToArr(curOperator)[1].split("").length >= 3 ? true : false;
+  } else {
+    return totalCountToArr("").length >= 3 ? true : false;
+  }
+};
```

Comment:
```suggestion
const isMaxLength = () => {
  const curOperator = totalCountToArr("").find((v) => isOperator(v));
  if (curOperator) {
    return totalCountToArr(curOperator)[1].split("").length >= 3;
  }
  return totalCountToArr("").length >= 3;
};
```
이 부분은 이렇게 표현할 수 있겠네요! 그리고 if로 진입시 바로 return 하기 때문에 else 구문은 불필요해보입니다.

Reply:
이 코드의 경우 영서님이 아닌 다른 사람이 봤을 때 이게 정확히 무엇에 대한 isMaxLength인지 가늠하기 힘들지 않을까요?

- 이 코드는 실제로 totalCount라는 dom의 길이가 maxLength인지 판단하고 있습니다.
- 그리고 maxLength는 3인데 함수 내부에서만 표현되고 있습니다.
- 즉, 이 함수의 이름만 봤을 때 무슨 용도인지 파악하기란 쉽지 않은 상태인거죠

이 과정은 `클린코드` 과정입니다.
- 내가 작성한 코드가 유지보수하기 쉬운가
- 유지보수 하기 쉽다는 것은
- 코드를 읽기 쉽다는 것
- 코드를 이해하기 쉽다는 것
- 내가 아닌 다른 사람도 이 코드를 보고 쉽게 수정할 수 있다는 것

그렇기 때문에 위에 대한 고민을 지속적으로 해보시는게 좋습니다 ㅎㅎ

Reply:
준일님 피드백을 받고서 많은 생각이 들었습니다. 
내가 아닌 다른사람이 봤을 때 무엇을 하는 함수인지 알기 쉬워야한다.
항상 고민을 하지만 다시볼때마다 아쉽기도하고 아직도 많이 어려운 부분인 것 같습니다.
피드백을 토대로 코드를 수정했지만 이 부분이 말끔하게 해결된건지는 아직 의문입니다 ㅠㅠ